payment-cards/pci-compliance/saqs.md

```diff
@@ -4,24 +4,34 @@ title: Self Assessment Questionnaires
 permalink: /payment-cards/pci-compliance/saqs/
 ---
 
-To be [PCI Compliant](/payment-cards/pci-compliance/) as a merchant, you'll need to fill out an SAQ, or Self Assessment Questionnaire.
+As a merchant, the least burdensome way to become [PCI Compliant](/payment-cards/pci-compliance/) is to fill out a Self Assessment Questionnaire (the next step up would be to hire a [QSA](/glossary/#qsa)).
 
 Most likely, the 2 SAQ's that are relevant to you if you accept payments online are [SAQ A](#saq-a) and [SAQ A-EP](#saq-a-ep). For a full list, see [Understanding the SAQs for PCI DSS version 3](https://www.pcisecuritystandards.org/documents/Understanding_SAQs_PCI_DSS_v3.pdf).
```

Comment:
for a full list and comparison? (I know this doesn't actually reside as part of your commit)